freqtrade/leverage/liquidation_price.py

```diff
@@ -0,0 +1,106 @@
+from freqtrade.enums import Collateral, TradingMode
+from freqtrade.exceptions import OperationalException
+
+
+def liquidation_price(
+    exchange_name: str,
+    trading_mode: TradingMode,
+    ** k
```

Comment:
Please don't use **kwargs in methods where the variable is mandatory.

Using explicit parameters (eventually with defaults if they're not always relevant) will allow mypy to help you detect potential problems.

Reply:
Different exchanges might need different arguments, I'm not sure yet, collateral isn't mandatory if not using leverage